css-brochure-form/styles.css

```diff
@@ -0,0 +1,81 @@
+fieldset {
+  background-color: #fff;
+  border-style: none;
+  line-height: 1.6;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+h2 {
+  color: #ee6e69;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.2rem;
+  text-transform: uppercase;
+  letter-spacing: 0.5px;
+  margin-top: 1.5rem;
+  margin-bottom: -0.2rem;
+}
+
+label {
+  color: #24292e;
+  margin: 10px;
+  padding: 0 0 30px 0;
+  font-size: 10px;
+  font-family: 'Brandon Grotesque', sans-serif;
+  letter-spacing: 0.1px;
+  font-weight: 1000;
+  text-transform: uppercase;
+}
+
+input {
+  padding: 8px 270px 8px 10px;
```

Comment:
Woah!!! That's a lot of padding! What is this being used for? Whatever it is, there's a better way to do it. Reach out if you have any questions!